count/Feature/PreSettingsFeature.swift

```diff
@@ -0,0 +1,281 @@
+                                GetMoreVoicesView()
+                            } label: {
+                                Text("Get more voices")
+                            }
+                            HStack {
+                                Text("Rate")
+                                    .onTapGesture(count: 2) {
+                                        viewStore.send(.rateLabelDoubleTapped)
+                                    }
+                                Slider(value: viewStore.$rawSpeechRate, in: viewStore.speechRateRange, step: 0.05) {
+                                    Text("Speech rate")
+                                } minimumValueLabel: {
+                                    Image(systemName: "tortoise")
+                                } maximumValueLabel: {
+                                    Image(systemName: "hare")
+                                }
+                            }
+                            HStack {
+                                Text("Pitch")
+                                    .onTapGesture(count: 2) {
+                                        viewStore.send(.pitchLabelDoubleTapped)
+                                    }
+                                Slider(value: viewStore.$rawPitchMultiplier, in: viewStore.pitchMultiplierRange, step: 0.05) {
+                                    Text("Pitch")
+                                } minimumValueLabel: {
+                                    Image(systemName: "dial.low")
+                                } maximumValueLabel: {
+                                    Image(systemName: "dial.high")
+                                }
+                            }
+                            Button {
+                                viewStore.send(.testVoiceButtonTapped)
+                            } label: {
+                                HStack(spacing: 10) {
+                                    Image(systemName: "person.wave.2")
+                                    Text("Test Voice")
+                                }
+                                .frame(maxWidth: .infinity, alignment: .center)
+                            }
+                        } else {
+                            NavigationLink {
+                                GetMoreVoicesView()
+                            } label: {
+                                HStack {
+                                    Text("Error: no voices found on device")
+                                        .foregroundStyle(Color.red)
+                                }
+                            }
+                        }
+                    } header: {
+                        Text("Voice Settings")
+                            .font(.subheadline)
+                    }
+                    .listRowBackground(Color(.systemGroupedBackground))
+                }
+                .scrollContentBackground(.hidden)
+                .background(Color(.tertiarySystemBackground))
```

Comment:
I had to decide between showing the default list style (and it clashing with the parent topics view) or reversing the colors. I chose the latter. See: https://hachyderm.io/@twocentstudios/111363205276091934